oceanmodes/test/test_modes.py

```diff
@@ -238,12 +238,12 @@ def test_N2_const_equal_spacing(self):
         #########
         # Analytical solution for Eady growth rate
         #########
-        growthEady = np.zeros((1, len(k))
+        growthEady = np.zeros((len(k), 1))
         for i in range(len(k)):
-            if (np.tanh(.5*k)**-1 - .5*k) * (.5*k - np.tanh(.5*k)) < 0:
+            if (np.tanh(.5*k[i])**-1 - .5*k[i]) * (.5*k[i] - np.tanh(.5*k[i])) < 0:
```

Comment:
The current repository (rabernat/oceanmodes) already has the code amended to:

for i in range(len(k)):
            if (np.tanh(.5*k[i])**-1 - .5*k[i]) * (.5*k[i] - np.tanh(.5*k[i])) < 0: